services/catalog/docker-compose-extra.yml

```diff
@@ -35,7 +35,7 @@ services:
     depends_on:
       - postgres
   director:
-    image: ${DOCKER_REGISTRY:-itisfoundation}/director:${DOCKER_IMAGE_TAG:-latest}
+    image: local/director:production
```

Comment:
is this change correct?

Reply:
This is for manual exploratory testing (i.e. ``make run-devel``)
I enforce to use the local copy instead of the one in the registry
It does not have an effect on the code